skyportal/handlers/api/classification.py

```diff
@@ -318,12 +329,15 @@ def get(self, obj_id):
         """
 
         obj = Obj.get_if_readable_by(obj_id, self.current_user)
+        include_redshift = self.get_query_argument("includeRedshift", False)
         if obj is None:
             return self.error('Invalid obj_id.')
 
         classifications = obj.get_classifications_readable_by(self.current_user)
         for classification in classifications:
             del classification.groups
+            if include_redshift:
+                classification.obj_data = {'redshift': obj.redshift}
```

Comment:
`obj_data` is not documented, does not contain all fields of Obj, and does not follow the naming conventions we use to refer to database models in nested output. The right way to go here would be to use `joinedload` along with the new permissions interface. But since that API refactor is still in progress, it's fine as a placeholder for now to  do:

```suggestion
            if include_obj:
                classification.obj = obj
```

Reply:
I had tried this earlier but it gave this error in api output
`{'status': 'error',
 'message': 'Circular reference detected',
 'data': {},
 'version': '0.9.dev0+git20210208.b7194a7'}`